components/navbar/navbar.css

```diff
@@ -28,6 +28,20 @@
     color:var(--headerbg);
 }
 
+.nav .search-container{
+    display: flex;
+    align-items: center;
+}
+
+.nav input{
+    padding: 0.4rem;
+    margin-top: 0.5rem;
```

Comment:
Preffered to use px rather than rems for values <=10.

Reply:
Okay noted.